sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/common/RequestInfo.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.common;
+
+import io.stargate.sgv2.docsapi.api.common.tenant.TenantResolver;
+import io.stargate.sgv2.docsapi.api.common.token.CassandraTokenResolver;
+import io.vertx.ext.web.RoutingContext;
+
+import javax.enterprise.context.RequestScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.ws.rs.core.SecurityContext;
+import java.util.Optional;
+
+/**
+ * The request information containing the tenant ID and the Cassandra key. This bean is @{@link RequestScoped}.
+ * <p>
+ * Uses the registered {@link TenantResolver} and {@link CassandraTokenResolver} to optionally resolve the tenant ID and the Cassandra token.
+ */
+@RequestScoped
+public class RequestInfo {
```

Comment:
Maybe `StargateRequestInfo`?

Reply:
Funny, I had it as `StargateRequestInfo` at the beginning, then switched, sure will revoke..